mlir/include/mlir/Dialect/Luminous/IR/LuminousDialect.td

```diff
@@ -32,9 +32,22 @@ def Luminous_Dialect : Dialect {
       return "luminous.container_module";
     }
 
-    static StringRef getFuncAttrName() {
-      return "luminous.function";
+    static StringRef getLuminousFuncAttrName() {
```

Comment:
we just use `getFuncAttrName()`.  The fact that it's luminous is clear from the namespace